### PR TITLE
Docs

### LINKED ISSUE #7: Add documentation.
Add documentation:

- [ ] all widgets.
- [ ] forms.

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

Fix #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,76 +38,74 @@ The package helps with implementing data entry forms.
...
+    config: [
+        'vertical' => require ThemePath::BOOTSTRAP5_VERTICAL,
+        'horizontal' => require ThemePath::BOOTSTRAP5_HORIZONTAL,
+    ],
+    defaultConfig: 'vertical',
+);
 ```
 
-### Mutation testing
+... and use `PureField` helper for create widgets:
```
Comment (samdark) [inline README.md:61]:
```suggestion
... and use `PureField` helper to create widgets:
```

File: src/Field/Base/InputData/InputDataInterface.php [hunk 2]
```diff
@@ -4,26 +4,50 @@
 
 namespace Yiisoft\Form\Field\Base\InputData;
 
+/**
+ * Represents input data.
```
Comment (samdark) [inline src/Field/Base/InputData/InputDataInterface.php:8]: Is that correct?

**Comment by samdark** [comment]
@vjik can you help to fill phpdoc for interfaces?

File: src/Field/Base/InputData/InputDataInterface.php (hunk 2 above)
Comment (arogachev) [reply src/Field/Base/InputData/InputDataInterface.php]: Updated in #325.

File: docs/guide/en/README.md [hunk 3]
```diff
@@ -1,38 +1,12 @@
...
 
-- [Creating Forms](creating-forms.md)
+- [Fields Configuration](fields-configuration.md)
+- [Common field properties](field-properties.md)
+- [Built-in fields](build-in-fields.md)
+- [Data input](data-input.md)
+- [Form field themes](field-themes.md)
+- [Creating and Using Custom Fields](creating-fields.md)
+- [Pure fields](pure-fields.md)
+- [Creating Forms](creating-forms.md) -- ???
```
[the remark this answers is not in the document]
Comment (arogachev) [reply docs/guide/en/README.md:10]: Yes, it was moved to form-model package.

File: README.md [hunk 4]
```diff
@@ -23,12 +36,74 @@ The package helps with implementing data entry forms.
...
+ThemeContainer::initialize(
+    config: [
+        'vertical' => require ThemePath::BOOTSTRAP5_VERTICAL,
+        'horizontal' => require ThemePath::BOOTSTRAP5_HORIZONTAL,
+    ],
+    defaultConfig: 'vertical',
+);
+```
+
+... and use `PureField` helper to create widgets:
```
Comment (samdark) [inline README.md:62]:
```suggestion
... and use `Field` helper to create widgets:
```

File: README.md [hunk 5]
```diff
@@ -23,12 +36,74 @@ The package helps with implementing data entry forms.
...
+
+```php
+use Yiisoft\Form\PureField;
+
+echo PureField::text('firstName', theme: 'horizontal')->label('First Name')->autofocus();
+echo PureField::text('lastName', theme: 'horizontal')->label('Last Name');
+echo PureField::select('sex')->label('Sex')->optionsData(['m' => 'Male', 'f' => 'Female'])->prompt('—');
+echo PureField::number('age')->label('Age')->hint('Please enter your age.');
+echo PureField::submitButton('Submit')->buttonClass('primary');
```
Comment (samdark) [inline README.md:71]: Field, not PureField.

File: docs/guide/en/field-themes.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    defaultConfig: 'vertical',
+);
+```
+
+Two themes are available out of the box:
+
+- Bootstrap 5 Horizontal,
+- Bootstrap 5 Vertical.
+
+TODO: describe `themes-preview`
```
Comment (samdark) [inline docs/guide/en/field-themes.md:26]: Should be done.

File: docs/guide/en/field-properties.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+# Common field properties
```
Comment (samdark) [inline docs/guide/en/field-properties.md:1]: Overall, this page looks a bit odd.

File: docs/guide/en/pure-fields.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
+# Pure fields
```
Comment (samdark) [inline docs/guide/en/pure-fields.md:1]: Needs to be adjusted since "pure" was removed.

**Comment by tomaszkane** [comment]
Will there be an example of creating and maintaining a form that contains more than one language version?
Something like `<input type="text" name="title[en]">` ?

**Comment by arogachev** [comment]
@tomaszkane Sounds like [cookbook](https://github.com/yiisoft/docs/tree/master/cookbook/en) might be a better place for this kind of documentation.

**Review by samdark — APPROVED** [review]

**Pull request merged**